.github/workflows/diff_report.yml

```diff
@@ -35,7 +35,15 @@ jobs:
       branch: ${{ steps.branch.outputs.ref }}
       
     steps:
-    
+     - uses: khan/pull-request-comment-trigger@master
+       name: React with rocket on run
+       with:
+         trigger: ','
+         reaction: rocket
+       env:
+         GITHUB_TOKEN: '${{ secrets.GITHUB_TOKEN }}'
+     - run: 'echo We print it here for this action to work'
```

Comment:
Previously there was no echo, just whitespace and it was working ok.
If this step is unavoidable, I suggest to bring whitespace back (plus comment in code) instead of printing some debug-like phrases, e.g.
```
# There have to be command, but we have none to execute, so leave it empty
run: ' '
```
Something like this

Reply:
It was suggested by Roman Sir in [here](https://github.com/checkstyle/checkstyle/pull/9189#discussion_r559747574). It was happening previously also but since it was working fine so, didn't pay attention to it.

Reply:
I believe it was mostly about if condition.
@romani why do you think that there should be some echo command? I do not see need in it, simple `run: ' '` will work